e2e/test/scenarios/cross-version/source/03-questions.cy.spec.js

```diff
@@ -86,8 +110,14 @@ it("should create questions", () => {
   visualize();
   cy.get("circle");
 
-  cy.findByTestId("viz-settings-button").click();
-  cy.icon("area").click();
+  cy.findByTestId("viz-type-button").click();
+  cy.findByTestId("Area-button").click();
+
+  // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
```

Comment:
the purpose of this rule is that we shouldn't  be writing unscoped text selectors. It makes the tests hard to read and potentially introduces flakes and ambiguities.

These tests might be a special case, where we can't add scoping elements for older versions though 😢